packages/truffle-debug-utils/index.js

```diff
@@ -251,6 +251,14 @@ var DebugUtils = {
     );
   },
 
+  formatPC: function(pc) {
+    let hex = pc.toString(16);
+    if (hex.length % 2 !== 0) {
+      hex = "0" + hex; //ensure even length
+    }
+    return "  PC = " + pc.toString() + " = 0x" + hex;
```

Comment:
is the assumption that anyone looking for this info will know what `PC` stands for?

Reply:
Basically, yeah.